### DISCUSSION TITLE
Does XML parser need in vlib.net.xml?

### DISCUSSION THREAD

**Discussion by vladimirmyshkovski** [vlang/v#14612] [Ideas]
I just started learning the `V` and the`vlib` libraries, and saw that a lot of things were missing.

I saw that there is only `HTML` parser, but there is no` XML` parser.

I would take on this task in order to study the language if someone considers it necessary.

**Comment by medvednikov** [comment]
Hi

Yes, it'd be great :)

Thanks

**Comment by nerdalytics** [comment]
Hi @vladimirmyshkovski, any progress?
